### PR TITLE
Update story

### PR DESCRIPTION
**Pull request by tomato3713**

### CODE REVIEW

File: momotaro.c [hunk 1]
```diff
@@ -18,12 +22,29 @@ int when(an *person, enum Status status){
...
+}
+
 int main(){
   long long ago;
   an old_man, old_woman;
   was_lived();
   old_man.go_to = FIREWOOD_COLLECTING;
   old_woman.go_to = WASHING;
   when(&old_woman, WASHING);
+  a momo = flow("kawa");
```
Comment (imadahirosi) [inline momotaro.c]: a peach flows through the river

File: momotaro.c [hunk 2]
```diff
@@ -18,12 +22,29 @@ int when(an *person, enum Status status){
...
+  old_woman.go_to = HOME;
+  an momotaro = cut(momo);
```
Comment (imadahirosi) [inline momotaro.c]: a baby come out from peach
baby was named "momotaro"

**Review by imadahirosi — CHANGES_REQUESTED** [review]
thank you for your pull request.
some code is not "readable English"

File: momotaro.c [hunk 3]
```diff
@@ -1,11 +1,15 @@
 #include<stdio.h>
```
Comment (imadahirosi) [inline momotaro.c:13]: update your fork.
now, parent repository source code has discription comment.
insert your name to "Authors" in discription.

**Review by imadahirosi — CHANGES_REQUESTED** [review]
update your fork.
now, parent repository source code has discription comment.
insert your name to "Authors" in discription.

**Comment by tomato3713** [comment]
Thanks for your reviews. 
I fixed not "readable english" code. and add my name to "Authors".

**Comment by tomato3713** [comment]
I deleted the a.exe.

**Pull request merged**